lib/cfg-lex.l

```diff
@@ -314,6 +314,7 @@ word	[^ \#'"\(\)\{\}\[\]\\;\r\n\t,|\.@:]
 <INITIAL,filterx>>         { return KW_TA_GT; }
 <INITIAL,filterx>===       { return KW_TAV_EQ; }
 <INITIAL,filterx>!==       { return KW_TAV_NE; }
+<INITIAL,filterx><<        { return KW_SHIFT_LEFT; }
```

Comment:
Just wondering, why don't we use something like `+=` and allow `+` too?

Looking at other languages, the "list append" and "dict extend" functionalities are usually implemented as functions or methods, but binary operators like `<<`, `+=`, etc. look better and feel intuitive in my opinion.

Reply:
Really good idea! Thanks :)

After talking about this IRL, I have changed this PR to use "+=", but I have a branch in progress which implements "=" and type cast wrapping "=", etc...

I think this PR is useful even without all the operator supports, so I think we can merge this as is, and I will open a new PR with the rest later.

Reply:
Thanks.